ci-operator/config/openshift/installer/master.json

```diff
@@ -34,6 +40,20 @@
         "bin": {"paths": [{"source_path": "/go/src/github.com/openshift/installer/tectonic", "destination_dir": "."}]},
         "root": {"as": ["build"]}
       }
+    },
+    {
+      "from": "base-smoke",
+      "to": "installer-bazel",
+      "dockerfile_path": "images/tectonic-installer/Dockerfile.ci",
```

Comment:
Do we really need this?  The Dockerfile is elderly ([Terraform 0.11.1][1]!).  It looks like it's building an image with the unpacked installer tarball, but I thought we were already building that somewhere.  I'll see if I can find it.

[1]: https://github.com/openshift/installer/blob/741d5cc084e2e22730268c71c6ddfc97fec19c62/images/tectonic-installer/Dockerfile.ci#L7

Reply:
> It looks like it's building an image with the unpacked installer tarball, but I thought we were already building that somewhere. I'll see if I can find it.

Ah, my notes on our existing publishing [are here][1], and point at `branch-ci-origin-installer-images` which we run in postsubmits.

> This is currently also above, used for "base" image.. I copied from there...

I'm trying to wrap my head around the image flow here.  Who builds base-smoke?

[1]: https://github.com/openshift/installer/pull/106#issue-206113988

Reply:
base-smoke is an IS in openshift namespace, it's created from the quay.io/coreos/origin-release:bazel image

images created here are pipeline tags, so like `registry.svc.ci.openshift.org/${NAMESPACE}/pipeline:installer-bazel`